src/command_modules/azure-cli-profile/azure/cli/command_modules/profile/_help.py

```diff
@@ -59,3 +59,8 @@
     short-summary: Show the details of a subscription.
     long-summary: If the subscription isn't specified, shows the details of the default subscription.
 """
+
+helps['account get-access-token'] = """
+    type: command
+    long-summary: provides the token for trusted utils to access your Azure subscriptions. The token will be valid for at least 5 minutes with the maximum at 60 minutes. If the subscription argument isn't specified, use the current account.
```

Comment:
'use the current account' -> 'the current account is used'.
My 2c anyway on this wording...